pbs_exporter.sh

```diff
@@ -23,8 +26,8 @@ source "$CREDENTIALS_DIRECTORY/creds"
 [[ -z "${PUSHGATEWAY_URL}" ]] && echo >&2 "PUSHGATEWAY_URL is empty. Aborting" && exit 1
 
 AUTH_HEADER="Authorization: PBSAPIToken=$PBS_API_TOKEN_NAME:$PBS_API_TOKEN"
-
-pbs_json=$($CURL --silent --fail --show-error --compressed --header "$AUTH_HEADER" "$PBS_URL/api2/json/status/datastore-usage")
+PBS_CURL_OPTIONS=($PBS_CURL_OPTIONS --silent --fail --show-error --compressed --header "$AUTH_HEADER")
```

Comment:
@ebuzzz which additional curl options are useful for your use case? 
If it's just the `--insecure` flag then I'd suggest creating a new configuration option such as `INSECURE_SSL_VERIFICATION` so that it is obvious that it results in a lower security level

Reply:
Reducing repetition by setting all common flags in one variable is a good idea tho

Reply:
For now, only insecure is needed for my use-case. But I could think of use for `--capath`, `--ciphers`. They won't be needed often, but sometimes internal certificates can be messy.

I'll let you decide. If you want a single option for the insecure, I'll modify it that way.

Reply:
We can also use both: `INSECURE_SSL_VERIFICATION` for both the PBS and PushGateway cURL calls. And a `CURL_OPTIONS`  for all the other options?

Reply:
I see, thanks for clarifying!
I think the 2 variable approach (INSECURE_SSL_VERIFICATION +CURL_OPTIONS) sounds good